osbs/conf.py

```diff
@@ -554,3 +565,22 @@ def get_platform_node_selector(self, platform):
                 nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_platform_descriptors(self):
```

Comment:
I think I was being overly prepared. I'll remove self.get_platform_descriptors since it isn't used anywhere anyway.